### PR TITLE
ADEN-7483 Fix integration with ad-products bidders

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: app/modules/ads/legacyModule.js [hunk 1]
```diff
@@ -122,7 +122,6 @@ class Ads {
 				'ext.wikia.adEngine.adLogicPageParams',
 				'ext.wikia.adEngine.config.mobile',
 				'ext.wikia.adEngine.context.slotsContext',
-				'ext.wikia.adEngine.lookup.a9',
```
Comment (fraszczakszymon) [inline app/modules/ads/legacyModule.js]: We can't remove that :(. Also we need to handle fvDelay on mobile without AE3

Comment (dmnsobczak) [reply app/modules/ads/legacyModule.js]: Restored. FVDelay will be implemented in separated ticket - it's a bigger change - for now A9 in FV will be disabled.

File: app/modules/ads/ad-context.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ export default {
 	bidders: {
 		enabled: false,
 		timeout: 2000,
+		disabledSlots: [],
```
Comment (fraszczakszymon) [inline app/modules/ads/ad-context.js]: https://github.com/Wikia/ad-products/pull/131/files#r209212784

**Review by fraszczakszymon — APPROVED** [review]

File: app/modules/ads/ad-context.js (hunk 2 above)
Comment (fraszczakszymon) [reply app/modules/ads/ad-context.js]: It stays empty on MW+AE3, right?

Comment (dmnsobczak) [reply app/modules/ads/ad-context.js]: Yep, it's not necessary.

**Pull request merged**